index.html

```diff
@@ -1,12 +1,192 @@
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Womens Conference</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="css/home.css" />
+    <script
+      src="https://kit.fontawesome.com/eb94992e2c.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <nav class="top-nav desktop-only">
+          <a href="" target="_blank">
+            <i class="fa-brands fa-facebook-f" aria-hidden="true"></i>
+            <span class="sr-only">Facebook</span>
+          </a>
+          <a href="" target="_blank">
+            <i class="fa-brands fa-twitter" aria-hidden="true"></i>
+            <span class="sr-only">Twitter</span>
+          </a>
+          <span class="color-3 font-2">English</span
+          ><span class="color-3 font-2">My Page</span
+          ><span class="color-3 font-2">Logout</span>
+        </nav>
+        <nav class="page-navigation">
+          <!-- mobile menu navigation overlay -->
+          <div id="mobile-menu-navigation" class="overlay mobile-only">
+            <a href="javascript:void(0)" class="close-btn">
+              <i class="fa-solid fa-xmark" aria-hidden="true"></i>
+              <span class="sr-only">Close Button</span>
+            </a>
+            <ul class="overlay-content font-2">
+              <li class="page-link">
+                <a href="about.html">About</a>
+              </li>
+              <li class="page-link">
+                <a href="javascript:void(0)">Sponsor</a>
+              </li>
+              <li class="page-link">
+                <a href="javascript:void(0)">Join</a>
+              </li>
+              <li class="page-link">
+                <a href="javascript:void(0)">Programs</a>
+              </li>
+              <li class="page-link">
+                <a href="javascript:void(0)">News</a>
+              </li>
+            </ul>
+          </div>
+          <!-- end of mobile navigation overlay -->
+          <span class="ham-container color-2 mobile-only"
+            ><img class="hamburger" src="images/hamburger.png" alt="Hamburger"
+          /></span>
+          <div class="desktop-nav desktop-only">
+            <span id="cc-Logo">
+              <a class="logo" href="index.html">
+                <img src="images/women-conference-Logo.jpg" alt="Logo" />
+              </a>
+            </span>
+            <ul class="desktop-nav-items color-2 font-2">
+              <li>
+                <a href="about.html">About</a>
+              </li>
+              <li>Program</li>
+              <li>Sponsor</li>
+              <li>Join</li>
+              <li>News</li>
```

Comment:
- [ ] Please wrap these elements in anchor tag <a>, They are menu options and supposed to be link.